code/network/multimsgs.cpp

```diff
@@ -4062,6 +4062,7 @@ void process_endgame_packet(ubyte * /*data*/, header *hinfo)
 
 		// make sure we're not already in the debrief state
 		if((gameseq_get_state() != GS_STATE_DEBRIEF) && (gameseq_get_state() != GS_STATE_MULTI_DOGFIGHT_DEBRIEF)){
+			Netgame.game_state = NETGAME_STATE_ENDGAME;
```

Comment:
Only the server should update `Netgame.game_state` so this is might cause issues since there is a small window in which it could be reset. I'm sure there are probably a few other places in the code where it get set like this, but it's best to try and avoid doing it.

I believe that `multi_warpout_all_players()` already sets a `Net_player` flag to indicate warpout so it might be safer to check that instead.

Reply:
Ok, changed, but I do still want to do a testing session before enabling it to be merged.